roles/icpc_host_backup/files/iptables.rules.new

```diff
@@ -0,0 +1,143 @@
+*filter
+:INPUT ACCEPT [0:0]
+:FORWARD DROP [0:0]
+:OUTPUT ACCEPT [0:0]
+
+# LOGGING / DEBUGGING
+-N accept-in-log
+-A accept-in-log -j LOG --log-level 4 --log-prefix "accept-in-log:"
+-A accept-in-log -j ACCEPT
+-N accept-out-log
+-A accept-out-log -j LOG --log-level 4 --log-prefix "accept-out-log:"
+-A accept-out-log -j ACCEPT
+-N drop-n-log
+-A drop-n-log -j LOG -m limit --limit 5/m --limit-burst 7 --log-level 4 --log-prefix "DENIED:"
+-A drop-n-log -j DROP
+
+# internal interface
+-A INPUT -i lo -j ACCEPT
+-A OUTPUT -o lo -j ACCEPT
+
+# open up server to server communication
+-A OUTPUT -d 10.3.3.208/29 -j ACCEPT
+-A INPUT -s 10.3.3.208/29 -j ACCEPT
+
+# ldap inbound
+-A INPUT -p tcp --sport 389 -s 10.3.3.210 -j ACCEPT
+-A OUTPUT -p tcp --dport 389 -d 10.3.3.210 -j ACCEPT
+-A INPUT -p tcp --sport 389 -s 10.3.3.211 -j ACCEPT
+-A OUTPUT -p tcp --dport 389 -d 10.3.3.211 -j ACCEPT
+-A INPUT -p tcp --sport 636 -s 10.3.3.210 -j ACCEPT
+-A OUTPUT -p tcp --dport 636 -d 10.3.3.210 -j ACCEPT
+-A INPUT -p tcp --sport 636 -s 10.3.3.211 -j ACCEPT
+-A OUTPUT -p tcp --dport 636 -d 10.3.3.211 -j ACCEPT
+
+# ldap outbound
+-A INPUT -p tcp --dport 189 -j ACCEPT
+-A OUTPUT -p tcp --sport 189 -j ACCEPT
+-A INPUT -p tcp --dport 636 -j ACCEPT
+-A OUTPUT -p tcp --sport 636 -j ACCEPT
+
+# DNS outbound
+-A INPUT -p tcp --sport 53 -j ACCEPT
+-A OUTPUT -p tcp --dport 53 -j ACCEPT
+-A INPUT -p udp --sport 53 -j ACCEPT
+-A OUTPUT -p udp --dport 53 -j ACCEPT
+
+# http outbound
+-A INPUT -p tcp --sport 80 -j ACCEPT
+-A OUTPUT -p tcp --dport 80 -j ACCEPT
+-A INPUT -p udp --sport 80 -j ACCEPT
+-A OUTPUT -p udp --dport 80 -j ACCEPT
+
+# https outbound
+-A INPUT -p tcp --sport 443 -j ACCEPT
+-A OUTPUT -p tcp --dport 443 -j ACCEPT
+-A INPUT -p udp --sport 443 -j ACCEPT
+-A OUTPUT -p udp --dport 443 -j ACCEPT
+
+# http
+-A OUTPUT -p tcp --dport 80 -d 10.3.3.208 -j ACCEPT
+-A INPUT -p tcp --sport 80 -s 10.3.3.208 -j ACCEPT
+
+# ntp in
+-A OUTPUT -p tcp --sport 123 -j ACCEPT
+-A OUTPUT -p udp --sport 123 -j ACCEPT
+-A INPUT -p tcp --dport 123 -j ACCEPT
+-A INPUT -p udp --dport 123 -j ACCEPT
+# ntp out
+-A OUTPUT -p tcp --dport 123 -j ACCEPT
+-A OUTPUT -p udp --dport 123 -j ACCEPT
+-A INPUT -p tcp --sport 123 -j ACCEPT
+-A INPUT -p udp --sport 123 -j ACCEPT
+
+# kattis uses http and (smtp or submission)
```

Comment:
Put this behind a flag? I dont see the reason to open this for this contest.

Reply:
good idea,  will move it to template.

Reply:
actually I don't need to talk to any ccs from the backup machines :)